cmd/spire-server/cli/bundle/bundle_test.go

```diff
@@ -76,7 +76,8 @@ func TestShow(t *testing.T) {
 			},
 			}
 
-			args := append(test.args, tt.args...)
+			args := test.args
```

Comment:
Hmm, I know that our code doesn't have a problem in practice, but I feel like this workaround is going against the spirit of the lint check. I think ideally we address the core of the issue which is to stop providing the appearance of evil around some suspicious slice appending. What if the `test` struct, instead of having an `args []string` member, stashed the `socketPath` and then provided function like so:
```
func (t *test) args(extra ...string) []string {
    return append([]string{"-socketPath", t.socketPath}, extra...)
}
```
And called like so:
```
rc := test.client.Run(test.args(tt.args...))
```
Then we can get out of the whole tricky situation.